tox.ini

```
@@ -43,9 +47,32 @@ commands =
     coverage combine
 
 [testenv:report]
-skip_install = true
+description= Generate coverage report
 dependency_groups =
     coverage
 commands =
     coverage report
     coverage xml
+
+[testenv:pyavd-utils-wheel]
+# Used as the package environment
+package_glob=pyavd-utils/dist/*
+
+[testenv:pyavd-utils]
+# The purpose of this environment is to have smaller footprint one for pyavd-utils only.
+# it is intended to be used we --installpkg from CLI supplying the already built wheel.
+description= Running pyavd-utils tests only
+# overwrite from default
+skip_install=false
+dependency_groups=
+
+package=external
+package_env=pyavd-utils-wheel
+
+deps=
+    pytest
```

For the warning, I would just add it here. I think the overhead is negligible and it keeps our configuration centralized in pyproject.toml.
```suggestion
    pytest
    # Adding unused pytest-asyncio to avoid warning because
    # of centralized pytest configuration for AVD repo.
    pytest-asyncio
```

---

added a comment in yoru suggestipon but yeah lets do it